src/prefect/server/events/stream.py

```diff
@@ -119,6 +125,10 @@ class Distributor:
 
     async def start(self):
         await start_distributor()
+        try:
+            await _distributor_task
+        except asyncio.CancelledError:
+            pass
```

Comment:
This is unrelated, but I noticed that it _looks_ like the distributor stops immediately after starting.  It's just because our `start` functions aren't supposed to return early